### DISCUSSION TITLE
Wix4 Conversion - Certificate element

### DISCUSSION THREAD

**Discussion by wmanning** [wixtoolset/issues#7338] [Questions]
Doing more conversions of my MSIs to Wix4. One MSI has a bunch of certificates..

I ran into two issues:
- The BinaryKey in Wix3 has changed to BinaryRef in Wix4.  The conversion leaves it as BinaryKey. Should this be changed to BinaryRef?  I see no reason why not.
- Overwrite attribute was allowed in Wix3 but not in Wix4.  This issue (https://github.com/wixtoolset/issues/issues/5703) says it never worked in Wix3.  Should this attribute be removed and a warning/error placed in the log?

Certificate Element
    wix3: https://wixtoolset.org/docs/v3/xsd/iis/certificate/
    wix4: https://wixtoolset.org/docs/schema/iis/certificate/

Wanted to make sure I didn't miss anything before creating an issue.

**Comment by robmen** [comment]
I think that is right.

Late, but right. 😄
